.eslintrc.js

```diff
@@ -342,6 +342,7 @@ module.exports = {
         'react/no-children-prop': 'off',
         'react/jsx-key': 'warn', // TODO - increase this into 'error' level
         'react-hooks/rules-of-hooks': 'error',
+        'consistent-return': 'off',
```

Comment:
This is the ESLint config block for React TypeScript code. The block for _all_ TypeScript is above, starting on line 135. This probably should go there instead, since the reason we're disabling this has nothing to do with React